dts/bindings/sensor/invensense,icp201xx.yaml

```diff
@@ -0,0 +1,51 @@
+# Copyright (c) 2025 TDK Invensense
+# SPDX-License-Identifier: Apache-2.0
+
+description: ICP201xx High Accuracy, Low Power, Barometric Pressure and Temperature Sensor IC
+
+compatible: "invensense,icp201xx"
+
+include: [sensor-device.yaml]
+
+properties:
+  int-gpios:
+    type: phandle-array
+    description: |
+      The INT signal default configuration is active-high. The
+      property value should ensure the flags properly describe the
+      signal that is presented to the driver.
+
+  op-mode:
+    type: string
+    default: mode0
```

Comment:
Need to explain in the description why the default value was selected
https://docs.zephyrproject.org/latest/build/dts/bindings-upstream.html#dt-bindings-default-rules